modules/shared/src/main/scala/org/tessellation/currency/dataApplication/package.scala

```diff
@@ -204,7 +236,18 @@ trait DataApplicationL0Service[F[_], D <: DataUpdate, DON <: DataOnChainState, D
 
 trait DataApplicationL1Service[F[_], D <: DataUpdate, DON <: DataOnChainState, DOF <: DataCalculatedState]
     extends DataApplicationService[F, D, DON, DOF]
-    with DataApplicationL1ContextualOps[F, D, DON, DOF]
+    with DataApplicationL1ContextualOps[F, D, DON, DOF] {
+
+  def requestDecoder(implicit a: Async[F]): EntityDecoder[F, NonEmptyList[Signed[DataTransaction]]] = {
```

Comment:
We've already used `A` for `Applicative`. I'd use `F` for an effect type as it's already a common pattern

Reply:
👍🏼